src/ansys/geometry/core/misc/checks.py

```diff
@@ -302,3 +303,48 @@ def check_type_all_elements_in_iterable(
     """
     for elem in input:
         check_type(elem, expected_type)
+
+
+def min_backend_version(method_version: str):
```

Comment:
I'm thinking... why not use 3 integers to define the backend version? major, minor an sp version, And we build the string on our own inside of it.

I'm just thinking of a wrongly-formatted decorator - string input is very flexible. However integers can only be integers. Anyway, what do you think?

Reply:
Agree, I prefer your suggestion.